soloconnect/settings.py

```diff
@@ -138,3 +138,11 @@
 LOGIN_REDIRECT_URL = "user_profile:view_profile"
 
 CRISPY_TEMPLATE_PACK = "bootstrap4"
+# Email configs
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_HOST = "smtp.gmail.com"
+EMAIL_USE_TLS = True
+EMAIL_USE_SSL = False
+EMAIL_HOST_USER = "soloconnect4@gmail.com"
+EMAIL_HOST_PASSWORD = "boyp dmed papx airi"
```

Comment:
Do we want to hardcode this here - we should ideally add it to AWS configuration and source it as an environment variable

Reply:
Yeah, we can do that. But this is not the real password, also, 2FA is turned on for this email account.